packages/components/app/styles/components/dialog.scss

```diff
@@ -0,0 +1,123 @@
+//
+// DIALOG
+//
+
+@use "../mixins/focus-ring" as *;
+
+.hds-dialog {
+  flex-direction: column;
+  padding: 0;
+  background: var(--token-color-surface-primary);
+  border: 1px solid rgba(59, 61, 69, 40%); // --token-color-border-strong;
+  border-radius: 10px;
+  box-shadow: var(--token-elevation-higher-box-shadow);
+
+  &[open] {
+    display: flex;
+  }
+
+  &::backdrop {
+    background: rgba(12, 12, 14, 50%); // --token-color-palette-neutral-700
+  }
+}
+
+.hds-dialog__header {
+  display: flex;
+  flex: none;
+  align-items: flex-start;
+  padding: 12px 24px;
+  border-top-left-radius: inherit;
+  border-top-right-radius: inherit;
+}
+
+.hds-dialog__icon {
+  flex: none;
+  margin-right: 16px;
+}
+
+.hds-dialog__title {
+  flex-grow: 1;
+  font-weight: var(--token-typography-font-weight-semibold);
+  font-size: var(--token-typography-display-200-font-size);
+  font-family: var(--token-typography-display-200-font-family);
+  line-height: var(--token-typography-display-200-line-height);
```

Comment:
better to use typography helper class names in the HTML/HBS element directly

Reply:
sure, will swap to class names

Reply:
Replaced these with class names